code/modules/mining/ores_coins.dm

```diff
@@ -304,18 +304,71 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 /obj/item/coin

```

Comment:
this file needs to go like this

```DM
/obj/item/coin
    stuff
    var/stuff

/obj/item/coin/proc1
    stuff

/obj/item/coin/proc2
    stuff

/obj/item/coin/subtype
    stuff
    var/stuff

/obj/item/coin/subtype2
    stuff

Reply:
put all `/obj/item/coin` procs above any subtypes

Reply:
ok